src/stepFunctions/sendYotiLetter.asl.json

```diff
@@ -12,12 +12,12 @@
       "Choices": [
         {
           "Variable": "$.body.pdf_preference",
-          "StringEquals": "email",
+          "StringEquals": "EMAIL_ONLY",
```

Comment:
These can be moved to constants and kept consistent across different lambdas using it.